src/Geometry/GMUGeometryRenderer.m

```diff
@@ -176,7 +176,7 @@ - (void)renderPoint:(GMUPoint *)point
         GMSMarker *strongMarker = weakMarker;
         GMSMapView *strongMap = weakMap;
         strongMarker.icon = image;
-        if (!_isMapCleared) {
+        if (!self->_isMapCleared) {
```

Comment:
Can we make a weakSelf outside the dispatch_async blocks and cast it back to a strongSelf here to avoid capturing self?